Dockerfile

```diff
@@ -150,6 +150,7 @@ ENV ALLOWED_HOSTS '["*"]'
 # Copy the application code over
 COPY cfgov ./cfgov/
 COPY static.in ./static.in/
+COPY . .
```

Comment:
I *think* on a read-through this is intended to copy the refresh-data.sh, test.sql.gz, and index.sh scripts, is that correct? Can we copy them explicitly, then layer cachingg will work for layers after this one.

Reply:
Yes that line copies over those three files.  You want them to each have their own `COPY` line?  Before or after `COPY . .`?

Reply:
They should have their own `COPY` lines replacing the `COPY . .` if we want to preserve layer caching after they're copied.